clients/client/src/main/java/org/projectnessie/client/http/HttpTreeClient.java

```diff
@@ -72,7 +77,11 @@ public Reference getReferenceByName(@NotNull GetReferenceParams params)
     return client
         .newRequest()
         .path("trees/tree/{ref}")
-        .queryParam("fetchAdditionalInfo", Boolean.toString(params.isFetchAdditionalInfo()))
+        .queryParam(
+            "view",
+            null == params.getViewOptions()
```

Comment:
Would it be possible if we move this block into a function since being used several times here?

```
            null == params.getViewOptions()
                ? ViewOption.BRIEF.toString()
                : params.getViewOptions().toString()
```

Reply:
yep, updated